Bottom: 96be31f76cc86e19e3d0bbe5775dbc2383b8da00
Top:    623a475191efa43ebea39d2b09a61e581a20047d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 23:15:08 +0100

GTK HID: Raise already open command window to top with ":" key

Previously, ":" only opened the window, but would not raise it
if already open.





---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 568d46e..09d8025 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1758,7 +1758,7 @@ LibraryChanged (int argc, char **argv, int x, int y)
 static int
 Command (int argc, char **argv, int x, int y)
 {
-  ghid_handle_user_command (FALSE);
+  ghid_handle_user_command (TRUE);
   return 0;
 }
